src/lib/EDSL.mli

```diff
@@ -23,31 +37,76 @@ val nop : unit t
 val if_then_else :
   bool t -> unit t -> unit t -> unit t
 val if_then : bool t -> unit t -> unit t
+
+val if_seq:
+  t:unit t list ->
+  ?e:unit t list ->
+  bool t ->
+  unit t
+(** [condition c ~t ~e] is an alternate API for {!if_then_else} (when
```

Comment:
The comment is not in sync with the documentation, judging from the `.ml`, `if_seq` instead of `condition`